draft-ietf-sframe-enc.md

```diff
@@ -1557,3 +1585,85 @@ The other values in the test vector are intermediate values provided to
 facilitate debugging of test failures.
 
 {::include test-vectors/sframe.md}
+
+# Acknowledgements
+{: numbered="false"}
+
+The authors wish to specially thank {{{Dr. Alex Gouaillard}}} as one of the early
+contributors to the document. His passion and energy were key to the design and
+development of SFrame.
+
+<!--[rfced] Terminology.
+
+a) FYI, we have added expansions for abbreviations upon first use per Section 3.6 of RFC 7322 ("RFC Style Guide"). Please review each expansion in the document carefully to ensure correctness.
+
+b) Please help us expand or define the following term:
+
+   SVC
+
+c) May we make the use of the acronym KID consistent?
+
+   Key ID (13) / KID (85)
+
+d) Please help us make the capitalization of the following terms consistent
+
+   counter (22) / Counter (9) - We note that Key ID is capitalized.
+                                Should CTR be used instead?
```

Comment:
When referring to the SFrame field, "Counter" should be used in the section defining the header format, and "CTR" elsewhere.  There are also several generic uses of "counter", which should be left as "counter".